babybear/.gitignore

```diff
@@ -0,0 +1,162 @@
+# This gitignore has been specially created by the WPILib team.
```

Comment:
babybear - Do we have consensus on the name of the robot?  At least from the Seniors (I think they get naming rights on the team).  If not them, how's about the captains?

Reply:
already changed to minibear